lib/inspec/runner.rb

```diff
@@ -134,7 +134,18 @@ def render_output(run_data)
       return if @conf["reporter"].nil?
 
       @conf["reporter"].each do |reporter|
-        result = Inspec::Reporters.render(reporter, run_data)
+        reporter_opts = {}
```

Comment:
Optionally the `run_data` could be directly modified here instead of passing options to the reporters. But I think there should be a distinction between the data that actually happened and the data that's reported